app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryEstimateButton.js

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux'
 
-export const CollapsedStoryEstimateButton = (props) => (
+export const CollapsedStoryEstimateButton = ({ project }) => (
```

Comment:
This component is still validating the props using `PropTypes.object`. Could you add a shape to it?